spec/mailers/user_mailer_spec.rb

```diff
@@ -791,6 +791,34 @@ def expect_email_body_to_have_help_and_contact_links
     end
   end
 
+  describe '#suspended_create_account' do
+    let(:mail) do
+      UserMailer.with(user: user, email_address: email_address).suspended_create_account
+    end
+
+    it_behaves_like 'a system email'
+    it_behaves_like 'an email that respects user email locale preference'
+
+    it 'sends to the specified email' do
+      expect(mail.to).to eq [email_address.email]
+    end
+
+    it 'renders the subject' do
+      expect(mail.subject).to eq t('user_mailer.suspended_create_account.subject')
+    end
+
+    it 'renders the body' do
+      expect(mail.html_part.body).
+        to have_content(
+          t(
+            'user_mailer.suspended_create_account.message',
```

Comment:
missing `app_name` here too:
```suggestion
            'user_mailer.suspended_create_account.message',
            app_name: APP_NAME,
```